js/src/wp-seo-wp-global-backport.js

```diff
@@ -0,0 +1,27 @@
+import * as importedData from "@wordpress/data";
+import * as importedElement from "@wordpress/element";
+import get from "lodash/get";
+
+let element = get( window, "wp.element", null );
+let data = get( window, "wp.data", null );
+
+// If Gutenberg is present we can just use their wp.element and wp.data.
```

Comment:
This comment doesn't really describe what is happening below, but what's happening on line 5 and 6. What's happening below is what happens when Gutenberg is not present. In that case, we use our own imported version of    Gutenberg.